buildSrc/src/main/kotlin/DependenciesExtensions.kt

```diff
@@ -42,6 +42,17 @@ fun Project.hiltDependencies() = dependencies {
     implementation(Libraries.Hilt.android)
 }
 
+fun Project.androidTestDependencies() = dependencies {
```

Comment:
Duvidazinha, no caso a chamada desse método(extension) já é feita dentro de um bloco `dependencies`, com isso, funcionaria fazer essa extension ser de `DependencyHandler`? Evitando assim multiplas chamadas de `dependencies`.
```suggestion
fun DependencyHandler.androidTestDependencies() {
```

Reply:
acredito que sim